### PR TITLE
community: Add explicit error message to Bedrock error output.

### PR DESCRIPTION
**Pull request by Mercurrent**
- **Description:** Propagate Bedrock errors into Langchain explicitly. Use-case: unset region error is hidden behind 'Could not load credentials...' message
  - **Issue:** [17654](https://github.com/langchain-ai/langchain/issues/17654)
  - **Dependencies:** None

### CODE REVIEW

File: libs/community/langchain_community/llms/bedrock.py [hunk 1]
```diff
@@ -335,11 +335,13 @@ def validate_environment(cls, values: Dict) -> Dict:
                 "Could not import boto3 python package. "
                 "Please install it with `pip install boto3`."
             )
+        except ValueError as e:
+            raise ValueError(f"Error raised by bedrock service: {e}")
```
Comment (baskaryan) [inline libs/community/langchain_community/llms/bedrock.py:339]: should we just reraise?
```
except ValueError as e:
    raise e
```

Comment (Mercurrent) [reply libs/community/langchain_community/llms/bedrock.py:339]: Wouldn't it be more transparent, that the error is not Langchain native, but side-service originated?

The exact error wording is "Did not find region_name, please add an environment variable `AWS_DEFAULT_REGION` which contains it, or pass `region_name` as a named parameter"

**Comment by Mercurrent** [comment]
Added error propagation to other classes affected same way. Added issue with description.

**Comment by Mercurrent** [comment]
@baskaryan friendly ping

**Pull request merged**